### PR TITLE
React Motivation

### PR DESCRIPTION
**Pull request by chasekap**

### CODE REVIEW

File: 07-react-motivation/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+##Why learn React?
```
Comment (evanzhong) [inline 07-react-motivation/README.md:2]: For some reason this isn't rendering properly on the README

image

File: 07-react-motivation/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+![DOM](ims/domimage.png)
+
+...this is for a very simple page. As you can imagine, for any real website, this would be a nightmare to manage, and would be a huge source of **bugs**.
+
+We'll be going more in-depth about this in the coming weeks, but React lets you avoid DOM management all together, and lets you manage how you display the page *as a function* (and functions are really good at reacting to change!). 
+
+This is **huge** in writing clean, bug-free applications. 
+
+#II. React is incredibly aesthetically pleasing (and saves you time)
```
Comment (evanzhong) [inline 07-react-motivation/README.md:38]: This also doesn't render properly I think
image

**Review by evanzhong — APPROVED** [review]
Looks good! Left some comments on the README, but non-blocking

File: 07-react-motivation/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+The psychological impact of having clean, concise code is often underappreciated. When your projects start to get big, navigating through your code can feel like running through a maze if it's messy and hard to understand. 
+Since programming is utimately a creative process, we want to make our work as smooth as possible. 
+
+Consider Twitter:
+```js
```
Comment (malsf21) [inline 07-react-motivation/README.md:44]: I think this should be \`\`\`html or \`\`\`jsx for code highlighting to work properly!

**Pull request merged**